lib/ansible.js

```diff
@@ -24,11 +25,20 @@ AbstractAnsibleCommand.prototype.exec = function(options) {
 
   var deferred = Q.defer();
   var processOptions = {};
-  if (options && options.cwd) {
+  if (!options) options = {};
+
+  if (options.cwd) {
     processOptions.cwd = options.cwd;
   }
 
+  // Make Ansible output unbuffered so the stdout and stderr `data` events 
+  // pickup output up more regularly
+  var unbuffered = (options.buffered) ? '' : 1;
+  processOptions.env = process.env;
+  _.extend( processOptions.env, { PYTHONUNBUFFERED: unbuffered } );
+
   var output = '';
+  debug('exec',this.commandName(), this.compileParams(), processOptions);
```

Comment:
Ah.. sorry, I was using that for my testing. It wasn't meant to make it into the commit.